packages/core/src/components/card/card.scss

```diff
@@ -11,15 +12,17 @@
   display: block;
 }
 
-:host(:hover) {
-  .dso-card-heading a {
-    color: card.$heading-anchor-hover-color;
-    text-decoration: underline;
-  }
+:host(:focus-within) {
+  background-color: colors.$grijs-5;
```

Comment:
Deze achtergrondkleur leent zich voor een semantische sass variable. We hebben immers twee andere componenten die ook deze achtergrondkleur moeten hebben (en belangrijker: ze moeten _dezelfde_ achtergrondkleur hebben).

Reply:
Gedaan